tests/VirtualContext.spec.ts

```diff
@@ -29,25 +30,36 @@ describe('virtual context', () => {
     expect(exports.require).toBeTruthy();
   });
 
+  it('should add browser mocks to the globals', () => {
+    let fakeBrowser = new BrowserWindow();
+
+    let results = Object.keys(fakeBrowser).filter(property => {
+      return !this.virtualContext.globals.hasOwnProperty(property);
+    });
+
+    expect(results.length).toBe(0);
+  });
+
   describe('with block scope scripts', () => {
 
     it('should run properly when strict mode is manually enabled', () => {
-      let filePath = path.normalize(__dirname + '/fixtures/virtual-context/fake-virtual-context-strict-mode.js');
+      let filePath = __dirname + '/fixtures/virtual-context/fake-virtual-context-strict-mode.js';

```

Comment:
This is not necessary. I intentionally removed it, since the VirtualContext / SandboxedRequire already handles that within it's module resolution.